invenio_app_ils/cli.py

```diff
@@ -669,11 +627,14 @@ def data(
 
     indexer = RecordIndexer()
 
+    with open('invenio_app_ils/vocabularies/data/tags.json') as f:
```

Comment:
this will not work when deployed, the path will be wrong. See https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/cli.py#L925